client/reader/lib/feed-display-helper/index.js

```diff
@@ -14,21 +14,6 @@ const exported = {
 		return url.replace( /^https?:\/\/(www\.)?/, '' ).replace( /\/$/, '' );
 	},
 
-	// Use either the site name, feed name or display URL for the feed name
-	getFeedTitle: function ( siteData, feedData, displayUrl ) {
-		let feedTitle;
-
-		if ( siteData && siteData.name ) {
-			feedTitle = siteData.name;
-		} else if ( feedData && feedData.name ) {
-			feedTitle = feedData.name;
-		} else {
-			feedTitle = displayUrl;
-		}
-
-		return feedTitle;
-	},
-
 	getFeedStreamUrl: function ( siteData, feedData ) {
```

Comment:
`getFeedStreamUrl` and `getSiteUrl` are also unused, the only function ever imported from this module is `formatUrlForDisplay`.

Reply:
Good spot, done as part of https://github.com/Automattic/wp-calypso/pull/86996/commits/d50632d25ac4c4d161bf74a8ccca5e385fd981bb